pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1351,7 +1351,7 @@
         desc: "Our customer support team is ready to answer your questions. Email us at [support@code.org](%{email_url}) or check out our support center, which offers useful guides and answers!"
         button: "Visit our support center"
     developed_in_partnership: "Developed in partnership with"
-  
+
```

Comment:
Completely unrelated to this PR but wondering if we can add some linting to prevent these whitespaces 🤔

Reply:
I think there already is one, I tend to only get these when others update this file. But I don't know if it's set up in the codebase or if [this VSCode extension](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) is doing it 😅

Reply:
Yeah my guess is we don't have it automated. (Obviously out of the scope of this PR and may not be worth adding at all, just thinking out loud.)